src/js/angular/core/directives/import-resource-tree/recursive.directive.js

```diff
@@ -0,0 +1,27 @@
+const modules = [];
+angular
+    .module('graphdb.framework.core.directives.recursive', modules)
+    .directive('recursive', recursive);
+
+recursive.$inject = ['$compile'];
+
+function recursive($compile) {
+    return {
+        restrict: "EACM",
+        priority: 100000,
```

Comment:
Why was this priority needed here?

Reply:
This is how i read the example of recursive using of a directive. This directive is not used now. I forgot to remove it.

Reply:
Removed